query-service-impl/src/test/java/org/hypertrace/core/query/service/pinot/QueryRequestToPinotSQLConverterTest.java

```diff
@@ -1013,6 +1004,19 @@ private Filter createTimeFilter(String columnName, Operator op, long value) {
     return Filter.newBuilder().setLhs(lhs).setOperator(op).setRhs(rhs).build();
   }
 
+  private Filter createLongTimeFilter(String columnName, Operator op, long value) {
```

Comment:
this is just a long filter, there's nothing time specific about it. That said, as with most of these private methods, I'd bet the test utils class already has this functionality ready to go, and this. can be removed.

Reply:
Refactoring some of the utility functions used here.

Reply:
resolved